frontend/src/App.tsx

```diff
@@ -4,6 +4,7 @@ import { mainTheme } from './configs/MuiConfig';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { lazy, Suspense } from 'react';
 import LandingPage from './components/Shared/LandingPage';
+import LogHours from './components/LogHours/LogHours';
```

Comment:
Could you import this with lazy import similar to the other import. This will decrease application load size.